src/App.js

```diff
@@ -22,25 +23,39 @@ export default function App() {
   const { push } = useHistory()
 
   return (
-    <Switch>
+    <>
+      <Switch>
+        <Route exact path="/">
+          <MainPage onHandleChange={changeCategory} />
+        </Route>
+        <Route path="/suggestions">
+          <Suggestions filteredMovies={filteredMovies} />
+        </Route>
+        <Route path="/addmovie">
+          <Toaster />
+          <MovieFormPage onAddMovie={addMovie} />
+        </Route>
+        <Route path="/movies">
+          <Toaster />
+          <MovieListPage
+            filteredMovies={filteredMovies}
+            onHandleChange={changeCategory}
+          />
+        </Route>
+      </Switch>
       <Route exact path="/">
```

Comment:
If you try it without the `exact` won't the navigation then render in each component? You have a lot of repetition here, maybe you could try and change it a bit so that you only have to set one route for the navigation.
You could also just render the navigation after the switch and then it should render in each component. At least I think that's possible.
```suggestion
      <Navigation />
```